Instructions-for-Reviewers.md

```diff
@@ -35,12 +35,13 @@ Simply visit this URL: https://github.com/pulls/assigned. It takes you to a list
      * **Important:** Please also ensure that, if a PR author files a TODO for future work, they assign themselves to it (or arrange for it to be assigned responsibly to someone else) and don't just abandon it. It should typically be the next thing they work on, unless there is a principled reason to agree otherwise. (In other words, PR authors shouldn't just file TODOs for future work and then abandon them.)
 
 
-      **Note**: To checkout branch BRANCH_NAME from committer COMMITTER_USERNAME to your local machine, run:
+      > **Note**
+      > To checkout branch BRANCH_NAME from committer COMMITTER_USERNAME to your local machine, run:
       ```
         git checkout -b {{COMMITTER_NAME}}-{{BRANCH_NAME}} develop
         git pull https://github.com/{{COMMITTER_NAME}}/oppia.git {{BRANCH_NAME}}
       ```
```

Comment:
This code block goes inside the note too. Right now it's not:
<img width="1028" alt="Screenshot 2023-01-24 at 20 58 34" src="https://user-images.githubusercontent.com/19878639/214463612-7872228e-d8dd-419c-b9ee-85d2aacd825a.png">

Reply:
My bad. Last time I didn't get this but fixed this time.